packages/nextjs/app/page.tsx

```diff
@@ -1,52 +1,193 @@
+    functionName: "allowList",
+    args: [connectedAddress],
+  });
+
+  const { data: IscheckedIn } = useScaffoldReadContract({
+    contractName: "BatchRegistry",
+    functionName: "yourContractAddress",
+    args: [connectedAddress],
+  });
+
   return (
-    <>
-      <div className="flex items-center flex-col flex-grow pt-10">
-        <div className="px-5">
-          <h1 className="text-center">
-            <span className="block text-2xl mb-2">Welcome to</span>
-            <span className="block text-4xl font-bold">Batch 9</span>
+    <div className="dark:bg-zinc-950 bg-zinc-200 dark:text-st_white text-st_background min-h-screen h-fit relative overflow-clip flex items-center justify-center">
+      <div className="absolute w-full h-full pointer-events-none">
+        <Image
+          src={blur_yellow}
+          alt="Blurred Yellow Background"
+          className="absolute -top-[10%] -left-[40%] w-200 h-200 blur-2xl"
+        />
+      </div>
+
+      <div className="absolute w-full h-full pointer-events-none">
+        <Image src={blur_blue} alt="Blurred Blue Background" className="absolute -bottom-[10%] -right-[40%] blur-2xl" />
+      </div>
+
+      <div className="max-w-7xl mx-auto flex flex-col items-center justify-center">
+        {/* Punks BG */}
+        <div className="overflow-clip">
+          <Image src={punks} alt="punks" />
+        </div>
+
+        {/* Header / SubHeader */}
+        <div className="mt-10 sm:mt-16 tracking-wider flex flex-col items-center justify-center">
+          <h1 className="font-semibold text-center text-4xl md:text-5xl lg:text-6xl xl:text-7xl">
+            Welcome to Batch 9 of
+            <span className="ml-3 font-bold">
+              <span className="text-st_cyan">Buidl</span>
+              <span className="text-st_yellow">Guidl</span>
+            </span>
           </h1>
-          <p className="text-center text-lg">Get started by taking a look at your batch GitHub repository.</p>
-          <p className="text-lg flex gap-2 justify-center">
-            <span className="font-bold">Checked in builders count:</span>
-            <span>To Be Implemented</span>
+          <p className="text-lg md:text-xl text-center lg:text-2xl xl:text-3xl">
+            Get started by taking a look at your batch GitHub repository.
           </p>
         </div>
 
-        <div className="flex-grow bg-base-300 w-full mt-16 px-8 py-12">
-          <div className="flex justify-center items-center gap-12 flex-col sm:flex-row">
-            <div className="flex flex-col bg-base-100 px-10 py-10 text-center items-center max-w-xs rounded-3xl">
-              <BugAntIcon className="h-8 w-8 fill-secondary" />
-              <p>
+        {/* Member Status */}
+        <div className="w-[16rem] sm:w-[34rem] text-center tracking-widest py-5 mt-6 text-st_cyan font-semibold rounded-xl border border-zinc-500 uppercase shadow-sm hover:bg-zinc-300 dark:hover:bg-zinc-800 transition-all">
```

Comment:
yeah you're correct, it's not necessary since it is not clickable. i just felt that it was a nice little touch on making the site feel 'alive' if you understand what i mean... the clickable links are differentiated by a cursor-pointer that appears on hover... if you still feel that the hover effect would cause any bad UX, let me know and i will remove it

Reply:
i got another idea while working on this... when the user is checked-in, the button turns into a link (with hover effect) which opens the corresponding builder page of the connected wallet... otherwise it is just a regular indicator (with no hover effect)

Reply:
Very nice idea! The only issue is that if a builder is signed but doesn't have a personal page, the link results in a 404 error.

Reply:
the best i can think of is to create a custom error page in /builders route. i can do that in another PR if you'd like..

Reply:
How about checking if the page exists (look at the error code yo are getting back fetching the page), and if it doesn't, displaying a message like 'Go build your builder page' instead? Alternatively, you could just disable the link.

Reply:
alright that's also a good idea. I'll do that